### ISSUE TITLE
Add onboarding card pattern

### ISSUE THREAD

**Issue by bmignano** [patternfly/patternfly-design#1154]
## Requesting new features, enhancements, or design changes to PatternFly
New feature request for an onboarding pattern. Today, I don't think we don't have any formal onboarding experience across products. This feature could also be a variation of / enhancement to the card component, but with additional specific guidelines regarding when to use it, where, content, etc. App Studio would like to use this pattern—ACM also has something similar today and CNV is looking into a similar pattern.

WIP design:
![app-view-card](https://user-images.githubusercontent.com/40238481/166699566-bb77afc8-4313-4893-b983-4c1058698826.png)
* The onboarding card can be used for day one getting started / onboarding help, or as a new feature highlight
* It will contain high level introductory information and an accompanying illustration
* The card is dismissible and cannot be reinvoked by the user
* The experience is per user

Questions to consider:
* If the user doesn't dismiss it, does it finally go away on its own after a set period of time?
* Should we have modal / carousel versions of this design?
* Is there ever a CTA in the card?

Modal option
![mvp-part2](https://user-images.githubusercontent.com/40238481/166700252-25809abb-55c1-4e65-9700-84b31f536e6f.png)
![mvp-part2](https://user-images.githubusercontent.com/40238481/166700272-3a744fc7-47e7-41ed-bb1c-d32df3460757.png)

Carousel option
![mvp-part2 copy](https://user-images.githubusercontent.com/40238481/166700341-4d7c9c33-d3bb-4688-b4d1-d3aca75a7cab.png)

**Comment by mmenestr** [comment]
@mcarrano Brie created this doc to document this: https://docs.google.com/document/d/1ugeSwI9t7iJ7HyQxuMhqCocWnI5okXGVZv5eARyP1Yc/edit#

If I remember correctly, the plan was to share this doc out to get feedback on which approach we should take to determine what to implement. 

I'm wondering if we should send this out as an email, or share it in the general channel for people to give feedback in their own time, and then maybe design share it at design share if we didn't get enough feedback by the time we have the next one. What do you think?

**Comment by mcarrano** [comment]
I suggest sending somethings out on Slack - the PF design channel, general channel in UXD or both - requesting feedback. Then maybe put this on the agenda at the next design share meeting.

**Comment by mcarrano** [comment]
@zhenpesky I'm assigning this issue out to you per out discussion yesterday. Feel free to add updates here as you work on this. Thanks for taking this on.

**Comment by zhenpesky** [comment]
@mcarrano here is the [updated link](https://docs.google.com/document/d/1ugeSwI9t7iJ7HyQxuMhqCocWnI5okXGVZv5eARyP1Yc/edit?usp=sharing) to the work.

**Comment by mcarrano** [comment]
@zhenpesky what's the latest on this?

**Comment by zhenpesky** [comment]
@mcarrano I updated the [document](https://docs.google.com/document/d/1ugeSwI9t7iJ7HyQxuMhqCocWnI5okXGVZv5eARyP1Yc/edit?usp=sharing) with the consolidated patterns with some guidelines and use cases. please take a look.

**Comment by mcarrano** [comment]
This is looking good @zhenpesky . I think this is a good candidate to add to the new Patterns section on the website. @andrew-ronaldson @mmenestr please also review and let me know your thoughts.

**Comment by mmenestr** [comment]
I agree it's a good start, would just want to see more info on **when** to use which one of these onboarding methods!

**Comment by zhenpesky** [comment]
To respond to this ticket, I have created an entry and documented the patterns for this project in the uxd-hub, please click [here to see the details](https://www.uxd-hub.com/entries/entry/onboarding-pattern).

**Comment by andrew-ronaldson** [comment]
Thanks for adding this @zhenpesky 
I will close this out and we'll refer to the Design Hub guidance when we start adding Patterns on our site.
